### ISSUE TITLE
NTR: Skin biopsy: collagen alpha5(IV) chain absent in epidermal basement membranes by immunohistochemistry (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9125]
New term request Skin biopsy: collagen alpha5(IV) chain absent in epidermal basement membranes by immunohistochemistry (elthat_100513011747):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6001026
- label: Reduced epidermal collagen IV alpah 5 chain staining
- parents: [Abnormal cutaneous collagen fibril morphology (HP:0031512)](https://hpo.jax.org/app/browse/term/HP:0031512)
- pmid: PMID:22921432
- def: Diminished or absent collagen alpha5(IV) chain staining pattern in the epidermal basement membrane.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9125
- synonyms:

**Comment by MickeySegal** [comment]
Typo.  Should be alpha 5

**Comment by pnrobinson** [comment]
fixed, thanks!
